CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 3.17 FATAL_ERROR)
+
+project(
+  vector_add_example
+  DESCRIPTION "Example using C++ Wrappers for the CUDA Driver API"
+  VERSION 0.0.1
+  HOMEPAGE_URL
+    https://github.com/nlesc-recruit/cudawrappers-usage-example-locally-installed
+  LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_BUILD_TYPE Release)
+
+# Use CMAKE_INSTALL_PREFIX when searching for libraries to run the installed
+# vector_add_example program.
+include(GNUInstallDirs)
+list(APPEND CMAKE_INSTALL_RPATH ${CMAKE_INSTALL_FULL_LIBDIR})
+
+find_package(cudawrappers 0.5.0 REQUIRED cu nvrtc)
+
+add_executable(vector_add_example src/vector_add.cpp)
+
+target_link_libraries(vector_add_example PRIVATE cudawrappers::cu
+                                                 cudawrappers::nvrtc)
+
+# Make it possible to #include cuda source code
+function(include_cuda_code target input_file)
```

Comment:
Would it be an idea to add this function in the cudawrappers CMakefile? So that also other examples could benefit?